_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -625,75 +667,71 @@ The metrics are defined as follows:
-This may work when the future is predictable, but not when it is complex and uncertain.
+| Official      | &#37; of Total Reqs | Mandatory Procedures (4) reqs[^13] |
+|---------------------------------------------------------------------|
+| ***CIO (with CDO)***  | ***84&#37;***             | ***229***       |
+| Service       | 10&#37;             | 0                             |
+| Cyber Security| 6&#37;              | 74                            |
 
-This complexity and uncertainty warrants organizations to move towards a more empirical cycle of trying, observing, and correcting.
-It is not just the execution phase of a project that needs to be ”agile”, it's the project's scope itself.
-The IT Project is the iteration step the organization is looking for as a means to become agile.
-This requires reviewing the relationship ESDC has with the use of technology and the interactions between IT teams.
+The current methods of managing IT investments is guided by the following ESDC financial policy instrument: [Policy on Project and Programme Management (PPPM)](https://gpp-ppm.service.gc.ca/sites/pwa/ESDCKnowledgeRepository/All%20Documents/Policy%20on%20Project%20and%20Programme%20Management.pdf). This is the Policy that produced the [Standard on Project Management](https://gpp-ppm.service.gc.ca/sites/pwa/ESDCKnowledgeRepository/All%20Documents/Standard%20on$20Project%20Management.pdf) which describes the key requirements for ESDC personnel operating in a project environment. The current standard seek to manage risks and costs with advanced planning and rigid plan execution. The gating approach to project management requires project teams to seek permission to continue. This permission is granted when sufficient evidence of due diligence and an accurate picture of the future is provided. This may work when the future is predictable, but not when it is complex and uncertain. Its adverse affect is promoting feature bloating as the burden of going through the process of seeking permission before proceeding makes it a deterrent to deploy small changes.
 
-This reality around the need to reduce risks and revisit the relationship stakeholders have with technology, is well explained in:
+Moving to a highly integrated, complex, and uncertain world that is Digital warrant us to move towards a more empirical cycle of trying, observing, and course correcting. The relationship between the Planning and Execution Phase becomes cyclical. It's through execution that we gain the necessary empirical evidence to inform our Planning estimates and decide on course corrections. Recent software development practices, mainly Cloud and DevOps, have permitted us to move towards this approach. IT no longer requires lengthy wait time to procure servers, to develop large code bases, and to seek large capital investments for infrastructure. Cloud has commoditized IT infrastructure enabling ESDC to rapidly develop, test, and deploy software.
```

Comment:
"...warrants us to". (add an s to warrant). But i would suggest changing to 'requires us to' as I feel like that is more easily understood.

And further down "..IT no longer requires lengthy wait times...' (add an s to time)